TOKEN_GUIDE.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Build a Token Generator with Astro
+description: Learn how you can generate tokens using SSR frameworks like Astro.
+---
+
+I love building websites with Astro. It's my favorite web framework, and I don't want to use any other one. However, I must stress test it and ensure Astro is flexible enough to fully commit.
```

Comment:
I think you should open with the second sentence and this sentence should come after.

> Astro is created to be the perfect use case for a content-heavy website. The opposite of that is a backend, and in this guide, we will build a backend token generator to be used for Agora video calls. I love building websites with Astro. It's my favorite web framework, and I don't want to use any other one. However, I must stress test it and ensure Astro is flexible enough to fully commit.

Reply:
Why? I don't disagree, but curious on your reasoning.

Reply:
The introductory sentences feel a bit disjointed from the topic of the guide. The statement  "I love building websites with Astro" feels like it explains why you're using Astro while the following sentences better set the stage for the guide, where you're using Astro as more of a backend than just SSR